rust/main/agents/relayer/src/relayer.rs

```diff
@@ -326,16 +326,21 @@ impl BaseAgent for Relayer {
         if let Some(tokio_console_server) = self.tokio_console_server.take() {
             let console_server = tokio::task::Builder::new()
                 .name("tokio_console_server")
-                .spawn(TaskMonitor::instrument(&task_monitor.clone(), async move {
-                    info!("Starting tokio console server");
-                    if let Err(e) = tokio_console_server.serve().await {
-                        error!(error=?e, "Tokio console server failed to start");
+                .spawn(TaskMonitor::instrument(
+                    &task_monitor.clone(),
+                    async move {
+                        info!("Starting tokio console server");
+                        if let Err(e) = tokio_console_server.serve().await {
+                            error!(error=?e, "Tokio console server failed to start");
+                        }
                     }
-                }))
+                    .instrument(info_span!("Tokio console server")),
+                ))
                 .expect("spawning tokio task from Builder is infallible");
-            tasks.push(console_server.instrument(info_span!("Tokio console server")));
+            tasks.push(console_server);
         }
-        let sender = BroadcastSender::new(ENDPOINT_MESSAGES_QUEUE_SIZE);
+        let sender: BroadcastSender<relayer_server::MessageRetryRequest> =
+            BroadcastSender::new(ENDPOINT_MESSAGES_QUEUE_SIZE);
```

Comment:
It looks like specifying the type is not necessary.

```suggestion
        let sender = BroadcastSender::new(ENDPOINT_MESSAGES_QUEUE_SIZE);
```